homeassistant/components/ipma/weather.py

```diff
@@ -87,31 +89,6 @@
 )
 
 
-async def async_setup_platform(
-    hass: HomeAssistant,
-    config: ConfigType,
-    async_add_entities: AddEntitiesCallback,
-    discovery_info: DiscoveryInfoType | None = None,
-) -> None:
-    """Set up the ipma platform.
-
-    Deprecated.
-    """
-    _LOGGER.warning("Loading IPMA via platform config is deprecated")
-
-    latitude = config.get(CONF_LATITUDE, hass.config.latitude)
-    longitude = config.get(CONF_LONGITUDE, hass.config.longitude)
-
-    if None in (latitude, longitude):
-        _LOGGER.error("Latitude or longitude not set in Home Assistant config")
-        return
-
-    api = await async_get_api(hass)
-    location = await async_get_location(hass, api, latitude, longitude)
-
-    async_add_entities([IPMAWeather(location, api, config)], True)
```

Comment:
It seems like this never triggered an import?
Should we still do that first?
What about raising a repairs issue?

Reply:
config_flow was added so long ago that there was no migrate code at the time :)

I've addressed the comment by adding `async_step_import`